src/app/admin/page.js

```diff
@@ -5,6 +5,6 @@ const AdminApp = dynamic(() => import('./adminApp'), {
   ssr: false,
 });
 
-export default function Admin() {
+export default function Page() {
```

Comment:
🟢 is there any particular reason for changing name? 🤔

Reply:
I think it would be nice to have a consistent name for page components

Reply:
i think that is subjective thing, since everywhere i saw in examples pages are named differently https://github.com/vercel/vercel/blob/main/examples/nextjs/app/page.tsx

but since its not named import i don't mind there be anything